source/Irrlicht/CMeshSceneNodeInstanced.cpp

```diff
@@ -484,7 +484,7 @@ void CMeshSceneNodeInstanced::RecullInstances()
             reinterpret_cast<uint32_t&>(lodCullingPointMesh->getMaterial().MaterialTypeParam) = i*gpuLoDsPerPass;
             reinterpret_cast<uint32_t&>(lodCullingPointMesh->getMaterial().MaterialTypeParam2) = i*gpuLoDsPerPass+gpuLoDsPerPass-1;
             driver->setMaterial(lodCullingPointMesh->getMaterial());
-            driver->beginTransformFeedback(xfb[i],lodCullingPointMesh->getMaterial().MaterialType,asset::EPT_POINTS);
+            //driver->beginTransformFeedback(xfb[i],lodCullingPointMesh->getMaterial().MaterialType,asset::EPT_POINTS);
```

Comment:
don't comment out, use
```
#ifndef NEW_SHADERS
#endif
```

and #define NEW_SHADERS in irr compile config.

Reply:
so what should i do with transform feedback? i mean this `NEW_MESHES` is not a solution

Reply:
is transform feedback non-optionally used in irrbaw somewhere behind the scenes? i have no idea about transform feedback

Reply:
@Crisspl TODO

Reply:
@Crisspl still a TODO but maybe in another branch/PR

Reply:
@Crisspl TODO.

Reply:
@Crisspl still a TODO

Reply:
just uncommented it now because im not touching scene nodes now, everything there is broken now so i could #ifndef NEW_SHADERS just as well

Reply:
ok